marblerun/_media/swagger.json

```diff
@@ -110,7 +110,7 @@
         },
         "/secrets": {
             "get": {
-                "description": "Each requests allows specifying one or more secrets in the form of a query string, where each parameter `s` specifies one secret.\nA query string for the secrets `symmetric_key_shared` and `cert_shared` may look like the following:\n\n```\ns=symmetric_key_shared\u0026s=cert_shared\n```\n\nThis API endpoint only works when `Users` were defined in the manifest.\nThe user connects via mutual TLS using the user client certificate in the TLS Handshake.\nFor more information, look up [Managing secrets](../#/workflows/managing-secrets.md).\n\nExample for retrieving the secrets `symmetric_key_shared` and `cert_shared`:\n\n```bash\ncurl --cacert marblerun.crt --cert user_certificate.crt --key user_private.key https://$MARBLERUN/secrets?s=symmetric_key_shared\u0026s=cert_shared\n```",
+                "description": "Each requests allows specifying one or more secrets in the form of a query string, where each parameter `s` specifies one secret.\nA query string for the secrets `symmetricKeyShared` and `cert_shared` may look like the following:\n\n```\ns=symmetricKeyShared\u0026s=cert_shared\n```\n\nThis API endpoint only works when `Users` were defined in the manifest.\nThe user connects via mutual TLS using the user client certificate in the TLS Handshake.\nFor more information, look up [Managing secrets](../#/workflows/managing-secrets.md).\n\nExample for retrieving the secrets `symmetricKeyShared` and `cert_shared`:\n\n```bash\ncurl --cacert marblerun.crt --cert user_certificate.crt --key user_private.key https://$MARBLERUN/secrets?s=symmetricKeyShared\u0026s=cert_shared\n```",
```

Comment:
`swagger.json` is an auto generated file and should not be modified by hand. Future GitHub workflow (edgelesssys/marblerun#230) will automatically generate and validate the `edgelesssys/marblerun/swagger.json` file as well as copy this file into this repo and create a PR for the changes.